include/modem/modem_info.h

```diff
@@ -352,6 +354,15 @@ int modem_info_get_rsrp(int *val);
  */
 int modem_info_get_connectivity_stats(int *tx_kbytes, int *rx_kbytes);
 
+/**
+ * @brief Obtain the current band
+ *
+ * @param band_id id of the current band
+ * @return int 0 if operation was sucessful.
```

Comment:
Good catch. Also updated my doxygen setting to prevent the type from getting added:

```json
"doxdocgen.generic.returnTemplate": "@return"
```

instead of the default `"@return" {type}`